src/components/Related/DomainsOnProtein/DomainsOnProteinLoaded/index.tsx

```diff
@@ -13,11 +13,19 @@ const ProteinViewer = loadable({
   loading: null,
 });
 
+// 0A017SEX7 is a good example
 const UNDERSCORE = /_/g;
 const FIRST_IN_ORDER = [
   'representative_domains',
-  'secondary_structure',
+  'representative_families', // coming soon
+  'variants',
+  'disordered_regions', // data coming from (?)
+  'residues',
+];
+
+const LASTS_IN_ORDER = [
   'family',
+  'secondary_structure',
```

Comment:
`LASTS_IN_ORDER` is to force some tracks to be at the end. Tracks that are neither in `FIRST_IN_ORDER` nor in `LASTS_IN_ORDER` will be in the middle, which is I think the desired behaviour.